examples/hosts/app-integration/external-data/README.md

```diff
@@ -80,7 +69,15 @@ Task - This is the unit that can be edited and attributed to an author. It is al
 
 TaskList - This can be compared to a "board" that holds all of the tasks. It is the larger visible entity to show up in a component within the app. The app stores two types of TaskLists - a "draft" version and a "saved version". More on these below.
 
+TaskList - This can be compared to a "board" that holds all of the tasks. It is the larger visible entity to show up in a component within the app. The app stores two types of TaskLists - a "draft" version and a "saved version". More on these below.
+
+ITaskData - This is similar to Task except that it is the External Data Services model of the task.
+
+ITaskListData - This is similar to TaskList except that it is the External Data Services model of the taskList.
+
+<!-- prettier-ignore-start -->
```

Comment:
Is there a specific reason we have to ignore prettier here?

Reply:
It's complaining about the non-capitalized f for fluid in the SharedMap link (https://fluidframework.com/docs/data-structures/map/)

Reply:
Wait, we do have a markdown linter!? Why does it force us to put four spaces between the dash and the text in a list item and why doesn't it force blank lines before and after headers? 😡

Sorry for the tangent.

Reply:
Ooo you're right -- okay I was able to remove this!